tests/pkg_elk/main.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the Elk Javascript engine
+ *
+ *              See README at https://github.com/cesanta/elk
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+#include <stdio.h>
+#include "elk.h"
+
+/* C function that adds two numbers. Will be called from JS */
+int sum(int a, int b) {
+  return a + b;
+}
+
+int main(void) {
+  char mem[256];
+  struct js *js = js_create(mem, sizeof(mem));  /* Create JS instance */
+  jsval_t v = js_import(js, sum, "iii");        /* Import C function "sum" */
```

Comment:
"iii" is for "takes two integers, returns an integer?"

Reply:
@benpicco correct. First letter describes return value, subsequent letters describe function arguments.

Reply:
Maybe we can expand the comment here to help users:

/* Import C function "sum" as taking two 'i'ntegers and returning an 'integer' */

Or is it

/* Import C function "sum" as returning an 'integer' and taking two 'i'ntegers */